### PR TITLE
Update README with migration instructions

### PR DESCRIPTION
**Pull request by msaraiva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,62 @@ For further information regarding installation, including how to quickly get sta
...
+Compile the dependencies:
+
+```
+mix clean && mix deps.get && mix deps.compile
+```
+Run the converter:
+```
+mix Surface.ConvertSyntax
+```
+
```
Comment (miguel-s) [inline README.md:158]: Maybe one last step?

Compile your app:
```
mix compile
```

**Review by miguel-s — APPROVED** [review]
LGTM! Just some tiny things 🙂

**Comment by msaraiva** [comment]
@miguel-s thanks for the review. I updated the text with your suggestions along with a couple of other adjustments. I also moved the "Limitations of the converter" up so the user can be aware of them before running it.

**Comment by msaraiva** [comment]
As the migration instructions gets improved, it's starting to pollute the `README.md` file a lot. I'll move it to its own file `MIGRATING.md`

File: MIGRATING.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  dependency are the same decribed in this guide.
+
+## Steps to run the converter
+
+Update your `.formatter` informing about `.sface` files and any additional folder where you might have any component
+to be converted:
+
+```
+[
+  surface_inputs: ["{lib,test,priv/catalogue}/**/*.{ex,exs,sface}"],
```
Comment (Malian) [inline MIGRATING.md:35]: Adding this line in my `.formatter.exs` it results in the following error when I run the converter:

```
** (ErlangError) Erlang error: {:badpattern, :missing_delimiter}
    (stdlib 3.15) filelib.erl:63: :filelib.wildcard/2
    (elixir 1.12.0) lib/path.ex:655: Path.wildcard/2
    lib/mix/tasks/surface/convert_syntax.ex:184: anonymous fn/4 in Mix.Tasks.Surface.ConvertSyntax.expand_dot_inputs/2
    (elixir 1.12.0) lib/enum.ex:2356: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/mix/tasks/surface/convert_syntax.ex:183: Mix.Tasks.Surface.ConvertSyntax.expand_dot_inputs/2
    lib/mix/tasks/surface/convert_syntax.ex:155: Mix.Tasks.Surface.ConvertSyntax.expand_args/3
    lib/mix/tasks/surface/convert_syntax.ex:98: Mix.Tasks.Surface.ConvertSyntax.run/1
    (mix 1.12.0) lib/mix/task.ex:394: anonymous fn/3 in Mix.Task.run_task/3
```

Using the following command works as expected:
```
mix surface.convert_syntax "lib/**/*.{ex,exs,sface}" 
```

Comment (Malian) [reply MIGRATING.md:35]: The issue is with the `priv/catalogue`. If you remove it, it works.

I do not know how to escape `/` inside `{}`, but the following proposition works
```suggestion
  surface_inputs: ["{lib,test,}/**/*.{ex,exs,sface}", "priv/catalogue/**/*.{ex,exs,sface}"],
```

**Comment by miguel-s** [comment]
I found the tiniest issue in the converter with my project, the indentation was slightly wrong in this case:

#### Original

```elixir
def render(assigns) do
  ~H"""
  <div class={{
    "inline-flex items-baseline px-2.5 py-0.5 rounded-full text-sm font-medium md:mt-2 lg:mt-0",
    "bg-green-100 text-green-800": @color == :green,
    "bg-red-100 text-red-800": @color == :red
  }}>
    ...
  </div>
  """
end
```

#### Converted

```elixir
def render(assigns) do
  ~F"""
  <div class={
    "inline-flex items-baseline px-2.5 py-0.5 rounded-full text-sm font-medium md:mt-2 lg:mt-0",
    "bg-green-100 text-green-800": @color == :green,
    "bg-red-100 text-red-800": @color == :red
 }> <- missing one whitespace
    ...
  </div>
  """
end
```

**Comment by msaraiva** [comment]
@miguel-s yeah, I need to make a small change in the converter.

**Pull request merged**